components/Footer.tsx

```diff
@@ -4,21 +4,26 @@ import catLady3 from '../assets/cat-lady3.png';
 import catLady3Web from '../assets/cat-lady3W.png';
 import catLady4 from '../assets/cat-lady4.png';
 import catLady4Web from '../assets/cat-lady4W.png';
+import catLady5 from '../assets/cat-lady5.png';
 import catLady5Web from '../assets/cat-lady5W.png';
 import { useNucaTheme as useTheme } from '../hooks/useNucaTheme';
-import { isSmallScreen } from '../utils/helperFunc';
+import { isSmallMobileScreen } from '../utils/helperFunc';
 
 const getFooterStyles = (_theme: NucaCustomTheme) =>
   StyleSheet.create({
     imageView: {
       marginTop: 32,
+      flex: 1,
     },
     image: {
+      alignSelf: 'center',
+      position: 'absolute',
+      bottom: isSmallMobileScreen() ? '-20%' : 0,
```

Comment:
can we avoid using negative offsets?

Reply:
Given that we have this issue when using only `bottom: 0`, I'm afraid that it is not possible (using `top` doesn't make sense, as we attempt to position the banner relative to the bottom of the screen, rather than the top):

<img width="364" alt="spatiu" src="https://github.com/crafting-software/nuca-mobile/assets/32801760/ee1034a1-3b65-4345-a520-3fc043d883d3">


The problem is that the banner is positioned a bit too far away from the bottom of the screen, one some devices.